ketcher-autotests/tests/utils/canvas/helpers.ts

```diff
@@ -319,3 +319,34 @@ export async function addMonomerToCenterOfCanvas(
   await clickInTheMiddleOfTheScreen(page);
   await selectRectangleSelectionTool(page);
 }
+
+export async function addPeptideOnCanvas(page: Page, peptideId: string) {
+  await page.getByTestId(peptideId).click();
+  await clickInTheMiddleOfTheScreen(page);
+}
+
+export async function takePeptideLibraryScreenshot(
+  page: Page,
+  options?: { masks?: Locator[]; maxDiffPixelRatio?: number },
+) {
+  const maxTimeout = 3000;
+  const editor = page.locator('[class*="MuiBox-root"]');
+  await waitForRender(page, emptyFunction, maxTimeout);
+  await expect(editor).toHaveScreenshot({
+    mask: options?.masks,
+    maxDiffPixelRatio: options?.maxDiffPixelRatio,
+  });
+}
+
+export async function takeLibrariesScreenshot(
+  page: Page,
+  options?: { masks?: Locator[]; maxDiffPixelRatio?: number },
+) {
+  const maxTimeout = 3000;
+  const editor = page.locator('[class*="shown monomer-library"]');
```

Comment:
I propose to create a more generic function for taking screenshots of specific element. We already have a bunch of functions which do the same job and the only difference is – test id